### PR TITLE
Doc:fix some mistake in Split_Learning_for_bank_marketing.ipynb

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
I fix some mistake in Split_Learning_for_bank_marketing.ipynb
Please review this PR of the update of the document. 
when the English content are accepted, I will follow [documentation-update-and-its-multilingual-version](https://github.com/secretflow/secretflow/blob/main/CONTRIBUTING.md#documentation-update-and-its-multilingual-version) supply the Chinese version of document.

### CODE REVIEW

**Review by zhangxingmeng — DISMISSED** [review]
LGTM

**Comment by zhouaihui** [comment]
Please resolve the conflict.

**Comment by zhouaihui** [comment]
I think we can merge #673 to this PR. One pr is enough.

**Comment by Mingbo-Lee** [comment]
> I think we can merge #673 to this PR. One pr is enough.
```
D:\WorkShop\secretflow\sl\secretflow>git fetch

D:\WorkShop\secretflow\sl\secretflow>git merge mingbo/fl-doc
merge: mingbo/fl-doc - not something we can merge
```
mayber we need accept two PR.

**Comment by zhouaihui** [comment]
Here merge means combine two PRs to one, not `git merge` - only one PR corrects the bank marketing doc.

File: docs/tutorial/Split_Learning_for_bank_marketing.ipynb [hunk 1]
```diff
@@ -1698,6 +1698,14 @@
     "single_model.evaluate(test_data, test_label, batch_size=128)"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Referring to the above visualization code, the training process of a local model can also be visualized."
```
Comment (zhouaihui) [inline docs/tutorial/Split_Learning_for_bank_marketing.ipynb:1706]: No CN for `Referring to the above visualization` in po file ?

Comment (Mingbo-Lee) [reply docs/tutorial/Split_Learning_for_bank_marketing.ipynb:1706]: I run
```
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/secretflow/fl/secretflow/docs$ sh update_po.sh
: not foundh: 2:
Running Sphinx v5.3.0
loading pickled environment... done
[autosummary] generating autosummary for: api/index.rst, awesome-pets/awesome-pets.md, awesome-pets/papers/applications/aml/attack_defense.md, awesome-pets/papers/applications/aml/poison.md, awesome-pets/papers/applications/multimedia/attack.md, awesome-pets/papers/applications/multimedia/defense.md, awesome-pets/papers/applications/multimedia/summary.md, awesome-pets/papers/applications/ppml/fl/Effectiveness.md, awesome-pets/papers/applications/ppml/fl/Efficiency.md, awesome-pets/papers/applications/ppml/fl/application.md, ..., tutorial/lr_with_spu.ipynb, tutorial/mix_lr.ipynb, tutorial/nn_with_spu.ipynb, tutorial/numpy_with_spu.ipynb, tutorial/practical_case_walkthrough_using_sf_with_spu.ipynb, tutorial/risk_control_scenario.ipynb, tutorial/split_learning_gnn.ipynb, tutorial/spu_basics.ipynb, tutorial/teeu.md, tutorial/teeu_xgboost.md
```
However，
```
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/practical_case_walkthrough_using_sf_with_spu.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/PSI_On_SPU.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/risk_control_scenario.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/SecureBoost.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/Split_Learning_for_bank_marketing.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/split_learning_gnn.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/spu_basics.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/teeu.po
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/teeu_xgboost.po
: not foundh: 4:
po files has been updated. Please update po files in locales folder.
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/secretflow/fl/secretflow/docs$
```
Here，
```
Not Changed: ./locales/zh_CN/LC_MESSAGES/tutorial/Split_Learning_for_bank_marketing.po
```

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**